pdh_windows.go

```diff
@@ -0,0 +1,120 @@
+package sigar
+
+/* These are convenience methods for extracting metrics from the Windows
+   Performance Data Helper (PDH) library */
+
+import (
+	"fmt"
+	"github.com/scalingdata/win"
+	"unsafe"
```

Comment:
Agreed, it should all be encapsulated. The library API is wrong, and requires the gnarly pointer arithmetic below. I'll push it all down into the library - it'll be harder to upstream, but this will be a lot cleaner.